src/details/ArborX_DetailsFDBSCANDenseBox.hpp

```diff
@@ -36,7 +36,7 @@ struct CountUpToN_DenseBox
   Kokkos::View<int *, MemorySpace> _counts;
   Primitives _primitives;
   DenseCellOffsets _dense_cell_offsets;
-  int _num_dense_cells;
+  unsigned _num_dense_cells;
```

Comment:
Why did you go this way rather than casting to `int` on the other side as suggested?

Reply:
It avoids casting in two places, as `_num_dense_cells` is initialized by `view.size()` which is unsigned, and then `k` is translated to signed.